app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -106,4 +106,44 @@ public function store(Request $request, $seatId = null, $backTo = null)
         // Redirect to the license page with error
         return redirect()->route('licenses.index')->with('error', trans('admin/licenses/message.checkin.error'));
     }
+
+    public function checkinCount($license_id){
+        $licenseSeats = LicenseSeat::where('license_id', '=', $license_id)
+            ->whereNotNull('assigned_to')
+            ->with('user')
+            ->get();
+
+        return('There are '.$licenseSeats->count().' seats checked out. Are you sure you want to check all of them in?');
+    }
+
+
+    public static function checkinLicense($license_id){
```

Comment:
I'd rename this to something like `checkinLicenseFromAll()` to avoid confusion. Also would love to see some docblocks here letting us know what each of these methods should do.